src/webapp/js/reducers/tables-data/tables_data_initial_state.js

```diff
@@ -8,4 +8,6 @@ export default {
   locationListPages: 1,
   userListPages: 1,
   roles: [],
+  rolesList: [],
```

Comment:
why do we have a list of roles and a "rolesList"?

Reply:
roles is the list of all roles used in selects and rolesList is the result of the search (in the roles table)

Reply:
let's change name of 'rolesList' to 'filteredRoles'